src/globals.cpp

```diff
@@ -48,6 +49,8 @@
 
 #include <atomic>
 #include <boost/lexical_cast.hpp>
+#include <boost/thread/tss.hpp> // for boost::thread_specific_ptr
```

Comment:
I don't see a use of thread_specific_ptr so remove this #include.  Also, don't use thread specific data in other project -- its unstable/difficult to use/link on some platforms.

Reply:
Weird. I don't remember adding this. Perhaps it's some rebase artifact. Anyhow, I removed it.